demo/shapes/fillings.html

```diff
@@ -2,29 +2,22 @@
 <html>
     <head>
         <meta charset="utf8"/>
-
-        <title>Gradients</title>
-
+        <title>Fillings</title>
```

Comment:
I think I would change the `<title>` to `Fills` and, `<script>` and filenames to `./src/fills.js + demo/shapes/fills.html`. 'Fillings' is a correct word, but not so usual in this context I think. I know `fill` and `gradient` are different when talking about the specific thing, and maybe you don't want to use the word 'fill', but I think in a general sense, a 'gradient' can be considered a type of 'fill' too. So `fills` makes sense I think. The rest of the docs look good.